infrastructure/variables.tf

```diff
@@ -0,0 +1,19 @@
+
+variable vpc_cidr_block {
+  description = "Base cider block for main vpc"
+  default = "172.17.0.0/16"
+}
+
+variable owner_code_name {
+  description = "Code name for owner"
+  default = "igork"
+}
+
+variable owner_label {
+  description = "Label for owner"
+  default = "Igor Kostin"
+}
+
+variable "vpc_version" {
```

Comment:
What is this used for, why we version vpc?

Reply:
I completely agree, the idea behind that was because this course has two practical parts, so the naming convention for the first was <Resource Name>_igork_01, and therefore for the second practical part should be 02, but since we have deleted the whole infrastructure from the previous part it is completely unnecessary and could be removed.